sprints/dashboard/utils.py

```diff
@@ -169,26 +176,31 @@ def _get_next_sprint(sprints: List[Sprint], previous_sprint_number: int, many=Fa
 
 
 def get_sprint_start_date(sprint: Sprint) -> str:
+    """Returns start date of the sprint."""
     if getattr(sprint, 'startDate', None):
         return sprint.startDate.split('T')[0]
 
     return extract_sprint_start_date_from_sprint_name(sprint.name)
 
 
 def get_sprint_end_date(sprint: Sprint, sprints: List[Sprint]) -> str:
+    """Returns end date of the sprint -1 day, because that's when the new sprint starts."""
     if getattr(sprint, 'endDate', None):
-        return sprint.endDate.split('T')[0]
+        date = sprint.endDate.split('T')[0]
+    else:
+        future_sprint = get_next_sprint(sprints, sprint)
+        date = get_sprint_start_date(future_sprint)
 
-    future_sprint = get_next_sprint(sprints, sprint)
-    return get_sprint_start_date(future_sprint)
+    return (parse(date) - timedelta(days=1)).strftime("%Y-%m-%d")
```

Comment:
This is because the sprints start and finish on the same day, right? I've seen my capacity at 86 hours but I think this fixes it, right?

Reply:
Yep, after we automated creating the sprints the commitment days began to overlap, because the day of ending current sprint and starting the new one is the same. This wasn't the case before, as the exact dates were not set.